package.json

```diff
@@ -56,6 +56,7 @@
         "azure-devops-node-api": "^11.2.0",
         "chai": "latest",
         "chalk": "^4.1.2",
+        "chokidar": "^3.5.3",
```

Comment:
The package-lock should be updated to match this.

Reply:
> Here's the kind of thing I was thinking as a patch (feel free to take it):
> 
> [...]

Would you use a plugin, or would passing a `WatchMode` object to `esbuild.build()` work just as well?

```js
esbuild.build({ ...await getOptions(), watch: { onRebuild(err, result) { ... } }, logLevel: "info" })
```